docs/source/ruletypes.rst

```diff
@@ -2051,6 +2053,8 @@ Example usage::
       ip: clientip
     http_post_static_payload:
       apikey: abc123
+    http_post_headers:
+      authorization: Basic 123dr3234
```

Comment:
A basic authorization example seems like a pretty loaded example for this property. However, it does clearly illustrate its utility beyond `foo: bar`.